### PR TITLE
team.html file

### PR DESCRIPTION
**Pull request by DarshitJain04**

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+            .container-fluid .iitj h5,p{
+                text-align:right;
+            }
+        }
+
+        .footer-copyright {
+            overflow: hidden;
+            color: rgba(255,255,255,0.6);
+            background-color: rgba(0,0,0,0.2);
+        }
```
Comment (killbotXD) [inline team.html]: Create a separate css file for this

File: team.html [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+	</script>
+    
+    <script>
+        $(".card-hover").hover(function(){
+        $(this).css("height", "350px");
+        }, function(){
+        $(this).css("height", "300px");
+    });
+        
+    console.log("team")
```
Comment (killbotXD) [inline team.html]: remove this

**Review by killbotXD — CHANGES_REQUESTED** [review]
Please do the requested changes

File: team.html (hunk 1 above)
Comment (DarshitJain04) [reply team.html]: Done

File: team.html (hunk 2 above)
Comment (DarshitJain04) [reply team.html]: Done

**Pull request merged**